src/screens/OtherProfile/index.js

```diff
@@ -603,7 +528,8 @@ const OtherProfile = () => {
   };
 
   const handleDataFeed = () => {
-    const isFeedsShown = !blockStatus.blocked && !blockStatus.blocker;
+    if (isProfileFetching) return [];
+    const isFeedsShown = !(isBlocking || isBlocked);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variables `blockStatus.blocked` and `blockStatus.blocker` have been replaced with `isBlocked` and `isBlocking` respectively. Ensure that these new variables provide the same functionality as the old ones.



<!-- This is an auto-generated comment by OSS CodeRabbit -->